### PR TITLE
Cloud module

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Basis of the cloud module. Further implementation will be done as soon as a design is ready
sign in

### CODE REVIEW

**Comment by RobFog** [comment]
"implimented" should read "implemented".

**Comment by igorkorsukov** [comment]
Just need to fix test execution

**Comment by Jojo-Schmitz** [comment]
Crashes doe to a failed assertion in an mtest, see https://travis-ci.org/github/musescore/MuseScore/jobs/708080562#L4779-L4783

File: mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                }
+            }
+
+            source: model.accountInfo.avatarUrl
+        }
+
+        StyledTextLabel {
+            anchors.verticalCenter: parent.verticalCenter
+
+            text: Boolean(model.accountInfo.userName) ? model.accountInfo.userName : "Sign in"
```
Comment (Eism) [inline mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml:50]: qsTrc

File: mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    }
+
+    MouseArea {
+        anchors.fill: parent
+        onClicked: model.logIn()
```
Comment (Eism) [inline mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml:56]: add braces

File: mu4/cloud/view/accountmodel.h [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+class AccountModel : public QObject, async::Asyncable
+{
+    Q_OBJECT
+
+    INJECT(account, IAccountController, accountController)
+
+    Q_PROPERTY(QVariant accountInfo READ accountInfo NOTIFY accountInfoChanged)
+
+public:
+    explicit AccountModel(QObject *parent = nullptr);
```
Comment (Eism) [inline mu4/cloud/view/accountmodel.h:40]: No space after a cast

File: mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml (hunk 1 above)
Comment (RomanPudashkin) [reply mu4/cloud/qml/MuseScore/Cloud/AccountInfo.qml:50]: It is a temporary solution and will be changed in the future

**Pull request merged**